studio/pages/404.tsx

```diff
@@ -11,62 +8,7 @@ const Error404: NextPage = ({}) => {
   const { ui } = useStore()
   const { theme } = ui
 
-  const [show404, setShow404] = useState<boolean>(false)
```

Comment:
@Isaiah-Hamilton can we revert these changes and let the dashboard team deal with it.
looks like you are removing an important useEffect here.